scripts/postinstall.sh

```diff
@@ -1,3 +1,15 @@
 #!/bin/bash
 
-cp node_modules/wikipedia-preview/dist/wikipedia-preview.production.js assets/js
+# compiled wikipedia-preview
+git submodule update
+cd wikipedia-preview
+git reset --hard
+npm i
+EXTERNAL_IMAGE_PATH='/wp-content/plugins/wikipediapreview-wordpress/assets/images' npm run build
```

Comment:
when I do this, i got the path `C:/Program Files/Git/wp-content/plugins/wikipediapreview-wordpress/assets/images/` because of my windows env, can you try it?

Reply:
it works for me